ray-operator/controllers/ray/common/pod.go

```diff
@@ -589,6 +589,9 @@ func setContainerEnvVars(pod *v1.Pod, rayContainerIndex int, rayNodeType rayiov1
 			}
 		}
 	}
+
+	// Set all fields in envs to the container.
+	container.Env = append(container.Env, envs...)
```

Comment:
If we already set envs in the podTemplate, this would be unnecessary.  this method set additional environments.

Reply:
fixed